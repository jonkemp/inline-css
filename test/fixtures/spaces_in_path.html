<html>
  <head>
    <link rel="stylesheet" href="spaces in path/Test.css">
  </head>
  <body>
    <p>hello, you are my <strong>favorite</strong> person</p></body></html>
