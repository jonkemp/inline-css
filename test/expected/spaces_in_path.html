<html>
  <head>
    
  </head>
  <body>
    <p>hello, you are my <strong style="color: blue;">favorite</strong> person</p>
  </body>
</html>
